### PR TITLE
Make wider use of async/await

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

### CODE REVIEW

File: src/sbvr-api/control-flow.ts [hunk 1]
```diff
@@ -17,12 +17,23 @@ export const liftP = <T, U>(fn: (v: T) => Resolvable<U>) => {
...
+) => Promise<Array<U | Error>>;
 
 // The settle version of `Promise.mapSeries`
-export const settleMapSeries: MappingFunction = (a, fn) => {
-	const runF = Bluebird.method(fn);
-	return Bluebird.mapSeries(a, _.flow(runF, wrap));
+export const settleMapSeries: MappingFunction = async <T, U>(
+	a: T[],
+	fn: (v: T) => Resolvable<U>,
+) => {
```
Comment (dfunckt) [inline src/sbvr-api/control-flow.ts:26]: How come you have to declare types both for the var and the arguments? Why isn't `MappingFunction` enough for TS to infer the argument types?

File: src/server-glue/server.ts [hunk 2]
```diff
@@ -84,13 +84,12 @@ if (!process.browser) {
 }
 
 export const initialised = Pinejs.init(app)
-	.then((configLoader) =>
-		Bluebird.all([
+	.then(async (configLoader) => {
+		await Bluebird.all([
```
Comment (dfunckt) [inline src/server-glue/server.ts:88]:
```suggestion
		await Promise.all([
```

File: src/sbvr-api/control-flow.ts (hunk 1 above)
Comment (Page-) [reply src/sbvr-api/control-flow.ts:26]: It's enough for it to infer the types but we won't have access to `U` for the `const results: Array<U | Error> = [];` below

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Pull request merged**